app/src/main/java/com/github/se/eventradar/map/MapViewModel.kt

```diff
@@ -1,53 +1,9 @@
 package com.github.se.eventradar.map
 
-import android.util.Log
 import androidx.lifecycle.ViewModel
-import com.github.se.eventradar.model.OverviewViewModel.Companion.getLocalDate
-import com.github.se.eventradar.model.location.getLocation
-import com.github.se.eventradar.model.todo.ToDo
-import com.github.se.eventradar.model.todo.ToDoList
-import com.github.se.eventradar.model.todo.getToDoStatus
 import com.google.firebase.firestore.ktx.firestore
 import com.google.firebase.ktx.Firebase
-import kotlinx.coroutines.flow.MutableStateFlow
-import kotlinx.coroutines.flow.StateFlow
 
 class MapViewModel : ViewModel() {
-  private val _uiState = MutableStateFlow(MapUiState())
-  val uiState: StateFlow<MapUiState> = _uiState
-
   private val db = Firebase.firestore
```

Comment:
nit: This line can be deleted with everything else.

Reply:
Agreed